Octokit/Models/Response/Reaction.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Octokit.Internal;
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    public enum ReactionType
+    {
+        [Parameter(Value = "+1")]
+        Plus1,
+        [Parameter(Value = "-1")]
+        Minus1,
+        [Parameter(Value = "laugh")]
```

Comment:
I **think** the default serialization will handle these standard values like Laugh, Confused, Heart etc and only the +1 and -1 would need a `[Parameter]` attribute?